### PR TITLE
Windup-3579: MTA/MTR CLI guides using OpenRewrite recipes

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
MTR 1.0, MTA 6.0
Resolves: https://issues.redhat.com/browse/WINDUP-3579
Preview:
MTA: https://deploy-preview-640--windup-documentation.netlify.app/docs/cli-guide/master/index.html#using-openrewrite-recipes_cli-guide
MTR: https://deploy-preview-640--windup-documentation.netlify.app/docs/cli-guide-mtr/master/index.html#using-openrewrite-recipes_cli-guide

### CODE REVIEW

File: docs/topics/using-openrewrite-recipes.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ For example, the OpenRewrite recipe `org.jboss.windup.JavaxToJakarta` renames im
 [source,terminal,subs="attributes+"]
 ----
 $ ./windup-cli --openrewrite --input </path/to/source/project> \
-  "-Drewrite.configLocation=<path/to/rewrite.yaml>  \
+  "-Drewrite.configLocation=<path/to/rewrite.yaml>"  \
```
Comment (stillalearner) [inline docs/topics/using-openrewrite-recipes.adoc:26]: line 44, 

  "DactiveRecipes=org.jboss.windup.JavaxInjectToJakartaInject, \
    org.jboss.windup.JavaxEjbToJakartaEjb"
    
    "-" hyphen missing

File: docs/topics/using-openrewrite-recipes.adoc [hunk 2]
```diff
@@ -54,6 +54,6 @@ You can include more than one recipe by specifying each in the `activeRecipes` p
 [source,terminal,subs="attributes+"]
 ----
 $ ../windup-cli --openrewrite --input </path/to/source/project> \
```
Comment (stillalearner) [inline docs/topics/using-openrewrite-recipes.adoc:56]: Extra (.) dot

**Comment by sbeskin-redhat** [comment]
@stillalearner 
Corrected, please check now

**Review by stillalearner — APPROVED** [review]
lgtm

File: docs/topics/available-openrewrite-recipes.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+.Available OpenRewrite recipes
+[cols="22%,26%,26%,26%", options="header"]
+|====
+|Migration path
+|Purpose
+|rewrite.configLocation
+|activeRecipes
+
+|Java EE to Jakarta EE
+|Replace import of javax packages with equivalent jakarta packages
```
Comment (RichardHoch) [inline docs/topics/available-openrewrite-recipes.adoc:17]: In the Purpose column, use `backticks` around filenames and locations. Likewise `javax` and `jakarta". See https://redhat-documentation.github.io/asciidoc-markup-conventions/.

**Comment by RichardHoch** [comment]
@sbeskin-redhat One small item (see files). Otherwise, LGTM. Once you address that, you can merge this PR.

File: docs/topics/available-openrewrite-recipes.adoc (hunk 3 above)
Comment (sbeskin-redhat) [reply docs/topics/available-openrewrite-recipes.adoc]: @RichardHoch  Thank you, corrected

**Pull request merged**